CURA-12428_Update Changelog (#20370)
Ultimaker/Cura: resources/texts/change_log.txt

[5.10]

* New features and improvements:
- UltiMaker S8 Support: Added compatibility for the UltiMaker S8 printer.
- Cheetah Gcode Flavor: Introduced a new Marlin-like Gcode flavor called Cheetah.
- SpaceMouse support: Navigate the view seamlessly with a 3D mouse, thanks to a collaboration with 3Dconnexion.
- Cloud Printing for Sketch Sprint: Enabled printing over cloud with Digital Factory for Sketch Sprint users.

> - Interlocking Settings: Moved Interlocking settings out of experimental into expert

Reviewer: Not sure what is meant by this; "experimental" is a settings category, "expert" is a setting visibility set. It looks like this should read that the settings were moved to the "Dual extrusion" category, and added to the "Expert" setting visibility preset.

Reviewer 2: Fixed in stable release notes

- Purge Lines And Unload Filament Post Processing Script: Added a Purge Lines and Unload Filament Post Processing Script contributed by @GregValiant and @HellAholic
- Thingiverse "Open in Cura" Button Linux Support: Enabled the "Open in Cura" button from Thingiverse to open files in Linux contributed by @hadess.
- Multitool Printer Configuration Options: Introduced 3 new configuration options in machine settings for multitool printers contributed by @TheSin-.
- Search and Replace Post-Processing Plug-In: Significantly improved the Search and Replace post-processing plug-in with features like replacing only the first instance, limiting search to a layer range, and ignoring start-up or ending G-code contributed by @GregValiant
- Enabled Relative extrusion (M82 and M83 commands) for Marlin-flavored GCode, contributed by @EmJay276

* New settings:
- Overhanging Wall Speeds, now gives you the ability to tune multiple values. Don’t forget to adjust the Overhanging Wall Angle to start using the setting. 
- Minimum Layer Time with Overhang and Minimum Overhang Segment Length: Fine-tune the minimum layer time for overhangs.

> - Distance to Walls: Finetune combing movements.

Reviewer 2: ```suggestion
- Inside Travel Avoid Distance: Finetune combing movements.
```

- Enable/Disable USB Printing: A hidden preference setting to indicate that you are using the printer over USB functionality. This setting lays the groundwork for automatically disabling USB printing in the next release when it’s not being used.

* Bug fixes:
- Resolved a crash that occurred when switching materials on Sovol printers.
- Random Seam no longer favors one side and not is truly random again
- Reduced the slicing time when no support needs to be generated 
- Fixed a bug where Seam on Vertex with a User Defined seam position was not working correctly.
- Gcode replacement with a single line of code no longer produces values in separate lines
- Setting names that become too long after translation are now truncated.
- Updated UltiMaker printer logos to align with the current style.
- The number of decimal places displayed for layer height in the top bar has been reduced.
- Fixed a bug that caused incorrect retracting and hopping on printers with more than 2 extruders.

> - Improved how fast the settings are loaded in the Settings Visibility window

Reviewer 2: ```suggestion
- Improved how fast the settings are loaded in the Settings Visibility window when scrolling
```

- Improved how disallowed areas are taken into account when arranging models on the buildplate

* Printer definitions, profiles, and materials:
- Introduced Visual Intents for the Sketch Sprint
- Introduced new Extra Fast and Draft profiles for the Sketch Sprint
- Introduced profiles for Sketch printers for Metallic PLA with improved surface quality (matte vs shiny)
- Introduce High Speed and High Speed Solid intents for Method, Method X, and Method XL
- Introduced PC ABS and PC ABS FR materials for Method X and Method XL
- Introduced Nylon Slide for UltiMaker S Series Printers
- Updated the Breakaway Build Volume Temperature for UltiMaker Factor 4
- Introduced Makerbot Replicator +
- Updated Voron2 printers to include TPU ASA and PVA, contributed by @WCEngineer
- Enabled Relative extrusion (M82 and M83 commands) for Marlin-flavored GCode, contributed by @EmJay276

> Cura 5.10 supports Mac OSX 12 Monterey or higher. This is because the tools that we use to create Cura builds no longer support previous versions of Mac OSX.

Reviewer 2: ```suggestion
Cura 5.10 supports macOS 12 Monterey or higher. This is because the tools that we use to create Cura builds no longer support previous versions of macOS.
```

Reviewer 2: In 2016, with the release of macOS 10.12 Sierra, the name was changed from OS X to macOS

[5.10]

* New features and improvements:
- UltiMaker S8 Support: Added compatibility for the UltiMaker S8 printer.
- Cheetah Gcode Flavor: Introduced a new Marlin-like Gcode flavor called Cheetah.

> - SpaceMouse support: Interact with your object in 3D thanks to a collaboration with 3Dconnexion

Reviewer 3: We already "interact with our objects in 3D" :smile: maybe an other description would be more appropriate ?

- Purge Lines And Unload Filament Post Processing Script: Added a Purge Lines and Unload Filament Post Processing Script contributed by @GregValiant and @HellAholic
- Thingiverse "Open in Cura" Button Linux Support: Enabled the "Open in Cura" button from Thingiverse to open files in Linux contributed by @hadess.
- Multitool Printer Configuration Options: Introduced 3 new configuration options in machine settings for multitool printers contributed by @TheSin-.
- Search and Replace Post-Processing Plug-In: Significantly improved the Search and Replace post-processing plug-in with features like replacing only the first instance, limiting search to a layer range, and ignoring start-up or ending G-code contributed by @GregValiant
- Enabled Relative extrusion (M82 and M83 commands) for Marlin-flavored GCode, contributed by @EmJay276

* New settings:
- Overhanging Wall Speeds, now gives you the ability to tune multiple values. Don’t forget to adjust the Overhanging Wall Angle to start using the setting. 
- Minimum Layer Time with Overhang and Minimum Overhang Segment Length: Fine-tune the minimum layer time for overhangs.
- Distance to Walls: Finetune combing movements.

> - Overhanging Wall Speeds: Fine-tune speeds based on different overhang percentages.

Reviewer 3: This is a duplicate of the first new setting

- Pressure Advance Factor Setting: New setting for machine definitions.
- You can now tune the Bottom Surface Skin, like you can tune the Top Surface Skin! You can now tune Extruder, Layers, Line Width, Pattern, Line Directions, Outer Wall Flow, Inner Wall(s) Flow, Flow, Outer Wall Speed, Inner Wall Speed, Skin Speed, Inner Wall Acceleration, Skin Acceleration, Outer Wall Jerk, Inner Wall Jerk, Skin Jerk, and Monotonic Bottom Surface Order

> - Enable/Disable USB Printing: A preference setting to indicate that you are using the printer over USB functionality. This setting lays the groundwork for automatically disabling USB printing in the next release when it’s not being used.

Reviewer 3: We should say that this is for now a hidden setting, or don't mention it, nothing is actually displayed anywhere so this text gives the impression that people can find a checkbox somewhere.

- The number of decimal places displayed for layer height in the top bar has been reduced.
- Fixed a bug that caused incorrect retracting and hopping on printers with more than 2 extruders.
- Improved how fast the settings are loaded in the Settings Visibility window
- Improved how disallowed areas are taken into account when arranging models on the buildplate

* Printer definitions, profiles, and materials:
- Introduced Visual Intents for the Sketch Sprint
- Introduced new Extra Fast and Draft profiles for the Sketch Sprint
- Introduced profiles for Sketch printers for Metallic PLA with improved surface quality (matte vs shiny)
- Introduce High Speed and High Speed Solid intents for Method, Method X, and Method XL
- Introduced PC ABS and PC ABS FR materials for Method X and Method XL
- Introduced Nylon Slide for UltiMaker S Series Printers
- Updated the Breakaway Build Volume Temperature for UltiMaker Factor 4
- Introduced Makerbot Replicator +
- Updated Voron2 printers to include TPU ASA and PVA, contributed by @WCEngineer
- Enabled Relative extrusion (M82 and M83 commands) for Marlin-flavored GCode, contributed by @EmJay276

> Cura 5.10 supports Mac OSX 12 Monterey or higher. This is because the tools that we use to create Cura builds no longer support these versions of Mac OSX.

Reviewer 3: The second sentence is confusing, should say "the tools that we use ... no longer support older versions of Mac OSX"
